import-export-cli/integration/testutils/devFirst_testUtils.go

```diff
@@ -172,6 +178,7 @@ func ValidateImportUpdateProjectNotAlreadyImported(t *testing.T, args *InitTestA
 	//Remove Created project and logout
 	t.Cleanup(func() {
 		base.RemoveDir(args.InitFlag)
+		base.WaitForIndexing()
```

Comment:
No need to wait here because we are only removing a directory and nothing related to indexing is taking place in APIM side.